.github/workflows/memcheck.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - name: Chunks data integrity during nodes churn
         run: cargo test --release -p sn_node --test data_with_churn -- --nocapture
         env:
-          TEST_DURATION_MINS: 15
+          TEST_DURATION_MINS: 5
```

Comment:
The number of nodes got churned is decided by the churning interval and the total testing time.
Previously, we have situation/bug that only got exposed when ALL original nodes got churned out.
So, the current testing time is to ensure that all original nodes got churned out.